pkg/operator/deploy/staticresources/master/deployment.yaml

```diff
@@ -3,6 +3,7 @@ kind: Deployment
 metadata:
   labels:
     app: aro-operator-master
+    version: GENERATED
```

Comment:
I don't think the changes to the yaml files are necessary

Reply:
@jim-minter it just makes it easier. Otherwise, We need to parse the image version. I just trying to get middle-ground over "parse the image" over, just read this one label with data.

Reply:
I'm +1 on adding the version label but you're doing that in the code -- there's no value to having a dummy value that's never used in the yaml as well.

Reply:
We do the same with Image. In theory, we could leave it `image: ` or drop it from the struct at all. I think `GENERATED` gives the reader of the code hint that this field is important in some way. 

on the other hand "GENERATED" is a valid value, so if code is removed it will slip into production...

Decisions decisions. :D Gonna remove with a hard hand.